src/manifest.json

```diff
@@ -22,25 +25,22 @@
         "css": ["templates/custom_styles.css"]
       }
     ],
-
     "permissions": [
-      "https://mycampus.hslu.ch/*",
-      "webRequest"
+        "https://mycampus.hslu.ch/*",
+        "webRequest",
+        "storage"
     ],
-
     "web_accessible_resources": [
-      "templates/grades_table.html",
-      "templates/credits_by_module_type_table.html",
-      "templates/custom_styles.css",
-      "data/modules_i.json",
-      "data/modules_ics.json",
-      "data/modules_wi.json"
+        "templates/grades_table.html",
```

Comment:
why the change from 2 spaces indent to 4 spaces in this file? And why hasn't it been done everywhere?
I think 2 spaces are enough for json, 4 would be ok too but a mix is not good.